server/libs/domain/src/system-config/system-config.core.ts

```diff
@@ -10,10 +10,12 @@ export type SystemConfigValidator = (config: SystemConfig) => void | Promise<voi
 const defaults: SystemConfig = Object.freeze({
   ffmpeg: {
     crf: '23',
+    threads: '2',
```

Comment:
Why is `2` the default? Wouldn't it be better to use ffmpeg's default of `0` which automatically determines the number of threads to use?

Reply:
There are many tasks running when photos and videos are added, so while FFmpeg chooses the best setting to max out transcoding, it can lead to thrashing between ML inference and thumbnail generation. Two threads seemed like a reasonable balance, but I can set the default to 0 if preferred.